frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/vm-settings-tab-state-update.ts

```diff
@@ -26,7 +26,7 @@ import { prefillVmTemplateUpdater } from './prefill-vm-template-state-update';
 export const selectUserTemplateOnLoadedUpdater = (options: UpdateOptions) => {
   const { id, dispatch, getState } = options;
   const state = getState();
-  if (options.changedCommonData.has(VMWizardProps.userTemplates)) {
+  if (!options.changedCommonData.has(VMWizardProps.userTemplates)) {
```

Comment:
can we check for the initialized first? As this updater will be checked only few seconds after opening and for the rest of life of the wizard the initialized prop should be enough

Reply:
:+1: